pkgs/development/ocaml-modules/janestreet/janePackage_0_16.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, fetchFromGitHub, buildDunePackage, defaultVersion ? "0.16.0" }:
+{ lib, fetchFromGitHub, buildDunePackage, defaultVersion ? "0.17.0" }:
```

Comment:
This is a non-starter. We must add the new version of Jane Street packages.

Reply:
My bad, I ran `nix-update` on ppx_string and did not realize it was part of the jane package set, dropping